### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by tmtocco**
This should be good for the nav bar for now; I'm not exactly sure how the navbar will show up on other pages in the future, but I think it'll generally look like this (with some possible design changes in the future)

### CODE REVIEW

File: Richmond_Library_App/templates/home.html [hunk 1]
```diff
@@ -7,16 +7,30 @@
     <link href="{% static 'home.css' %}" rel="stylesheet" type="text/css">
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <img src="static\images\mps_logo1.png" alt=" Logo">
```
Comment (TouKoua) [inline Richmond_Library_App/templates/home.html]: When I run the code, the image isn't loading for me. I fixed the issue by changing the src to follow the static directory. The one that worked for me was src="{% static 'images/mps_logo1.png' %}"

**Review by TouKoua — COMMENTED** [review]
Looks like a good start to me. Check my comment and consider if you want to fix it right away or not.

File: Richmond_Library_App/templates/home.html (hunk 1 above)
Comment (tmtocco) [reply Richmond_Library_App/templates/home.html]: Thanks for letting me know! I had some issues as well with the image showing up. What I had was working for me, so it must have just only worked for my side.  Just made the change and assume it should work now.

**Comment by nobodyname12** [comment]
Travis, are you able to pull in the latest changes so that this branch isn't behind?

**Review by ale202008 — APPROVED** [review]

**Review by TouKoua — APPROVED** [review]
cool. works on my end. good work

**Review by ale202008 — APPROVED** [review]

**Pull request merged**